constellation_base/apps.py

```diff
@@ -2,4 +2,7 @@
 
 
 class ConstellationBaseConfig(AppConfig):
-    name = 'ConstellationBase'
+    name = 'constellation_base'
+
+    def ready(self):
+        from . import signals
```

Comment:
You cannot use models until the ready function has been called.  Per the [Django docs](https://docs.djangoproject.com/en/1.11/ref/applications/#django.apps.AppConfig.ready), this is where that import goes.